gridded_etl_tools/util_funcs/projections.py

```diff
@@ -0,0 +1,64 @@
+import xarray as xr
+
+
+def assign_lambert_crs_to_grib(dataset: xr.Dataset, earth_radius: float, data_var: str) -> xr.Dataset:
```

Comment:
How hard would this be to generalize to other projected CRS, not just lambert? Could the lambert specific information be imported and filtered from a config file that could potentially hold other implementation details for other projected CRSes?